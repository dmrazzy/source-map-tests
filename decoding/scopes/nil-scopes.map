{
  "version": 3,
  "file": "nil-scopes.js",
  "sources": [
    "original_0.js",
    "original_1.js",
    "original_2.js",
    "original_3.js"
  ],
  "mappings": "",
  "names": [],
  "scopes": "A,A,A,A"
}