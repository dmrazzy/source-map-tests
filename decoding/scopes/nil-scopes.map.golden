{
  "file": "nil-scopes.js",
  "mappings": [],
  "sources": [
    {
      "url": "original_0.js",
      "content": null,
      "ignored": false,
      "scope": null
    },
    {
      "url": "original_1.js",
      "content": null,
      "ignored": false,
      "scope": null
    },
    {
      "url": "original_2.js",
      "content": null,
      "ignored": false,
      "scope": null
    },
    {
      "url": "original_3.js",
      "content": null,
      "ignored": false,
      "scope": null
    }
  ],
  "ranges": []
}